management-api-agent-common/src/main/java/io/k8ssandra/metrics/builder/CassandraMetricRegistryListener.java

```diff
@@ -373,20 +389,24 @@ private void setTimerFiller(
                   String.format("Unable to getOffsets for DSE, snapshotClass: %s", snapshotClass),
                   e);
             }
+          } else {
+            logger.debug(
+                String.format(
+                    "Unknown type for %s, wants: %s\n", proto.getMetricName(), snapshotClass));
           }
 
           // This can happen if histogram isn't EstimatedDecay or EstimatedHistogram
-          if (values.length != buckets.length) {
-            logger.debug(
+          if (values.length > buckets.length + 1 || values.length < buckets.length) {
```

Comment:
Question:
with this logic, it means that `values.length` can equal `buckets.length` OR `buckets.length + 1`. Is that what we want/need here?